pkg/operator/encryption/key_controller_encryption_test.go

```diff
@@ -38,7 +38,7 @@ func TestEncryptionKeyController(t *testing.T) {
 				schema.GroupResource{Group: "", Resource: "secrets"}: true,
 			},
 			targetNamespace: "kms",
-			expectedActions: []string{"list:secrets", "create:secrets"},
+			expectedActions: []string{"list:secrets:openshift-config-managed", "create:secrets:openshift-config-managed"},
```

Comment:
answering to your previous comment (https://github.com/enj/cluster-kube-apiserver-operator/pull/3#discussion_r317090343)
> I am thinking verb + resource string + resource name/ns always validated and optional full object validation. Ideally we do this generically in every test case.

I think we cannot get a resource's name as we are dealing with different kinds of actions and not all conform to a uniform way of getting the underlying resource. See: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/testing/actions.go

Reply:
But we should be able to get it on verbs like `Get`